### PR TITLE
[UI] Global Navigation Menu Tooltip Enhancement

### LINKED ISSUE #10203: [UI] Global Navigation Menu Tooltip Enhancement
### Current Behavior

The tooltip lacks consistency with other tooltips in Meshery. Additionally, the tooltip for the GitHub icon incorrectly displays `github[#142]`, as shown in the screenshot. This should also be removed.
### Desired Behavior

1. Make the font size consistent across tooltips.
2. Aim for a consistent background color for tooltips, ensuring it complements the existing UI and does not create visual conflicts.

### Screenshots/Logs

Screenshot 2024-02-04 at 12 39 33 AM
Screenshot 2024-02-04 at 12 39 24 AM
Screenshot 2024-02-04 at 12 39 18 AM

### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by ChetanPatil12**
**Notes for Reviewers**

- uexpected popup "Github [# 142]" removed 
- did nothing regarding tooltips issue since its already solved

This PR fixes #10203

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudhanshutech** [comment]
@ChetanPatil12 have not checked for dco?

**Comment by ChetanPatil12** [comment]
I don't know how to check for dco

File: ui/assets/icons/GithubIcon.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ const GithubIcon = (props) => {
       version="1.1"
       xmlns="http://www.w3.org/2000/svg"
     >
-      <title>github [#142]</title>
       <desc>Created with Sketch.</desc>
```
Comment (leecalcote) [inline ui/assets/icons/GithubIcon.js:18]:
```suggestion
```

**Review by leecalcote — APPROVED** [review]
Thanks!

**Pull request merged**